Src/Utils/lateinit.py

```
@@ -0,0 +1,18 @@
+    def __init__(self, method, *args, **kwargs):
+        self.__value = method
+        self.__args = args
+        self.__kwargs = kwargs
+
+
+    def __set_name__(self, owner, name):
+        self.owner = owner
+        self.name = name
+
+
+    def __get__(self, instance, owner: type = None):
```

Ruff (0.15.21)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)